File: src/codes/Users_Manual/stm/focus_1.py
~from pyautd3 import Hz, FociSTM
~import numpy as np
~foci = [np.array([0.0, 0.0, 0.0]), np.array([0.0, 0.0, 0.0])]
stm = FociSTM.from_sampling_config(1 * Hz, foci)

File: src/codes/Users_Manual/stm/focus_1.rs
# extern crate autd3;
# use autd3::prelude::*;
# #[allow(unused_variables)]
# fn main() -> Result<(), Box<dyn std::error::Error>> {
let stm = FociSTM::from_sampling_config(
    SamplingConfig::Freq(1 * Hz),
    [Vector3::zeros(), Vector3::zeros()],
);
# Ok(())
# }